dotnet3.5/S3/CreateBucketExample/CreateBucket/CreateBucket.cs

```diff
@@ -0,0 +1,110 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX - License - Identifier: Apache - 2.0
+// snippet-start:[S3.dotnet35.CreateBucket]
+
+using Amazon;
+using Amazon.S3;
+using Amazon.S3.Model;
+using Amazon.S3.Util;
+using System;
+using System.Threading.Tasks;
+
+namespace CreateBucket
+{
+    public class CreateBucket
+    {
+        // This example shows how to use Amazon Simple Storage Service (Amazon S3)
+        // to reate a new Amazon S3 bucket. The examples uses AWS .NET SDK 3.5 and
+        // .NET 5.0
+
+        // Specify your AWS Region (an example Region is shown).
+        private static readonly RegionEndpoint BUCKET_REGION = RegionEndpoint.USWest2;
+        private static IAmazonS3 _s3Client;
+
+        // Specify the name of the new bucket
+        private const string NEW_BUCKET_NAME = "doc-example-bucket";
+
+        static void Main()
+        {
+            _s3Client = new AmazonS3Client(BUCKET_REGION);
+            Console.WriteLine($"\nCreating a new bucket, named: {NEW_BUCKET_NAME}.");
+
+            CreatingBucketAsync(_s3Client, NEW_BUCKET_NAME).Wait();
+
+            var bucketLocation = FindBucketLocationAsync(_s3Client, NEW_BUCKET_NAME);
+
+            if(!String.IsNullOrEmpty(bucketLocation.ToString()))
+            {
+                Console.WriteLine($"\n\nBucket {NEW_BUCKET_NAME} successfully created.");
+            }
+            else
+            {
+                Console.WriteLine("The bucket does not exist.");
+            }
+        }
+
+        /// <summary>
+        /// Uses Amazon .NET Framework PutBucketAsync to create a new
```

Comment:
I don't think there is such a thing as the Amazon .NET Framework.

Reply:
As I said, fixed throughout. I double-checked the references to the SDK throughout all three projects.